c++17 lacks std::bind2nd
citra-qt currently needs SDL for audio and input

--- CMakeLists.txt.orig	2017-08-09 15:54:29 UTC
+++ CMakeLists.txt
@@ -89,7 +88,7 @@ message(STATUS "Target architecture: ${ARCHITECTURE}")
 # Configure compilation flags
 # ===========================
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if (NOT MSVC)
@@ -207,15 +207,15 @@ if (ENABLE_SDL2)
     else()
         find_package(SDL2 REQUIRED)
     endif()
-
-    if (SDL2_FOUND)
-        # TODO(yuriks): Make FindSDL2.cmake export an IMPORTED library instead
-        add_library(SDL2 INTERFACE)
-        target_link_libraries(SDL2 INTERFACE "${SDL2_LIBRARY}")
-        target_include_directories(SDL2 INTERFACE "${SDL2_INCLUDE_DIR}")
-    endif()
 else()
-    set(SDL2_FOUND NO)
+    find_package(SDL2)
+endif()
+
+if (SDL2_FOUND)
+  # TODO(yuriks): Make FindSDL2.cmake export an IMPORTED library instead
+  add_library(SDL2 INTERFACE)
+  target_link_libraries(SDL2 INTERFACE "${SDL2_LIBRARY}")
+  target_include_directories(SDL2 INTERFACE "${SDL2_INCLUDE_DIR}")
 endif()
 
 if (ENABLE_QT)
